<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>432</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>配置器</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Bing.ico</normaloff>Bing.ico</iconset>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>244</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_reset">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;重置以上设定&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>重设</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton_delete">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;删除开机启动配置&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_OK">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;配置开机启动并立即执行&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_1">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;选中复选框并在右侧输入限制壁纸的数量&lt;/span&gt;&lt;/p&gt;&lt;p&gt;下载近&lt;span style=&quot; font-weight:600;&quot;&gt;?n?&lt;/span&gt;天缺失的必应每日一图，并存档不为近&lt;span style=&quot; font-weight:600;&quot;&gt;?n?&lt;/span&gt;天的图片。&lt;/p&gt;&lt;p&gt;最多可下载近15天图片。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>限制壁纸数量</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;选中复选框并在右侧输入推送通知时长&lt;/span&gt;&lt;/p&gt;&lt;p&gt;不建议设置过小的数值。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>推送通知时长</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;选中复选框并在右侧输入出错重试次数&lt;/span&gt;&lt;/p&gt;&lt;p&gt;建议设置此项以避免网络状况不良导致的程序失效。&lt;/p&gt;&lt;p&gt;不建议设置过大的数值。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>出错重试次数</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;选中复选框并在右侧输入出错重试时间间隔&lt;/span&gt;&lt;/p&gt;&lt;p&gt;建议设置此项以避免网络状况不良导致的程序失效。&lt;/p&gt;&lt;p&gt;不建议设置过小或过大的数值。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>出错重试间隔</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QSpinBox" name="spinBox_1">
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>31</number>
      </property>
      <property name="value">
       <number>14</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QSpinBox" name="spinBox_2">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string>    秒</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QSpinBox" name="spinBox_3">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QSpinBox" name="spinBox_4">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string>    秒</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>必应每日一图</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="pushButton_exit">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_OK</tabstop>
  <tabstop>pushButton_reset</tabstop>
  <tabstop>pushButton_delete</tabstop>
  <tabstop>pushButton_exit</tabstop>
  <tabstop>checkBox_1</tabstop>
  <tabstop>spinBox_1</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>checkBox_3</tabstop>
  <tabstop>spinBox_3</tabstop>
  <tabstop>checkBox_4</tabstop>
  <tabstop>spinBox_4</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>OK_buttonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>Reset_buttonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_delete</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>Delete_buttonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_exit</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>Exit_buttonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_3</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>Sync_Retry_3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_4</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>Sync_Retry_4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>Update_toolTip()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OK_buttonClick()</slot>
  <slot>Reset_buttonClick()</slot>
  <slot>Delete_buttonClick()</slot>
  <slot>Exit_buttonClick()</slot>
  <slot>Sync_Retry_3()</slot>
  <slot>Sync_Retry_4()</slot>
  <slot>Update_toolTip()</slot>
 </slots>
</ui>
